src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -94,12 +95,11 @@ const AuditorRunTestForm: React.FC = () => {
   }, [resetForm]);
 
   const handleRepoFieldBlur = (event: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-    if (event.target.value !== form.getValues().repoUrl) {
+    if (event.target.value) {
```

Comment:
We need to make sure this check is not frequently triggered - if the value in field is not changed and simply clicked and blurred on.

Reply:
Done!